thunder/tests/opinfos.py

```diff
@@ -7970,3 +7970,34 @@ def item_sample_generator(op, device, dtype, requires_grad, **kwargs):
 
 
 opinfos.extend(memory_access_ops)
+
+
+def size_sample_generator(op, device, dtype, requires_grad, **kwargs):
+    make_t = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    shapes = (
+        (),
+        (1,),
+        (
+            2,
+            2,
+        ),
+        (0, 2, 1),
+        (2, 0, 1),
+    )
+
+    for shape in shapes:
+        t = make_t(shape)
+        yield SampleInput(t)
+
+        for d in range(len(shape)):
+            yield SampleInput(t, d)
+
+
+size_opinfo = OpInfo(
```

Comment:
Maybe we can put this here:

https://github.com/kshitij12345/lightning-thunder/blob/d30195d85ca6782a58553f0fe8677f930f1a9fa5/thunder/tests/opinfos.py#L565

as a tensor property?

Reply:
Right, I missed that. Thanks!